.gitignore

```diff
@@ -61,3 +61,4 @@ nosetests.xml
 udata.cfg
 stylemark/
 .parcel-cache
+.vscode/settings.json
```

Comment:
I'm ignoring it in `.git/info/exclude` because it depends on my dev environment.